examples/http_body.rs

```diff
@@ -18,10 +18,23 @@ use proxy_wasm::types::*;
 #[no_mangle]
 pub fn _start() {
     proxy_wasm::set_log_level(LogLevel::Trace);
-    proxy_wasm::set_http_context(|_, _| -> Box<dyn HttpContext> { Box::new(HttpBody) });
+    proxy_wasm::set_root_context(|_| -> Box<dyn RootContext> { Box::new(HttpBodyRoot) });
 }
 
 struct HttpBody;
+struct HttpBodyRoot;
+
+impl RootContext for HttpBodyRoot {
+    fn get_type(&self) -> ContextType {
+        ContextType::HttpContext
+    }
+
+    fn create_http_context(&self, _context_id: u32) -> Box<dyn HttpContext> {
+        Box::new(HttpBody)
+    }
+}
+
+impl Context for HttpBodyRoot {}
```

Comment:
Nit: move `impl Context` before `impl RootContext` to match order for `HttpBody`.

Reply:
done